### PR TITLE
[IMP] account: setup bank account configuration easy

### PR DESCRIPTION
**Pull request by rmu-odoo**
Description of the issue/feature this PR addresses:
Setup bank account configuration easy

Current behavior before PR:
User has to configure bank, bank sync, etc. on journal that make bit complicated for user

Desired behavior after PR is merged:
easy and full configuration of bank journal in one step.
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/pos_mercury/views/pos_mercury_views.xml [hunk 1]
```diff
@@ -78,6 +78,16 @@
                 </xpath>
             </field>
         </record>
+        <record id="view_account_bank_journal_pos_user_form" model="ir.ui.view">
```
Comment (mdi-odoo) [inline addons/pos_mercury/views/pos_mercury_views.xml]: Why you need this view? There is already a view for **account.journal**. See this: https://github.com/odoo/odoo/pull/15436/files#diff-a6c93aba12a2013af951703a88cbce10R71 .

File: addons/point_of_sale/views/account_statement_view.xml [hunk 2]
```diff
@@ -9,10 +9,22 @@
...
                         <group col="4" colspan="4">
                             <field name="journal_user"/>
-                            <field name="amount_authorized_diff"/>
+                            <field name="amount_authorized_diff" attrs="{'invisible': [('type','!=','cash')]}"/>
                         </group>
                     </page>
                 </xpath>
             </field>
         </record>
+        <record id="view_account_bank_journal_pos_user_form" model="ir.ui.view">
```
Comment (mdi-odoo) [inline addons/point_of_sale/views/account_statement_view.xml]: Why you need this view? There is already a view for **account.journal**. See this: https://github.com/odoo/odoo/pull/15436/files#diff-693b40947e7766b1d64a05686dfac8fcR3 .

File: addons/pos_mercury/views/pos_mercury_views.xml [hunk 3]
```diff
@@ -78,6 +78,16 @@
...
+            <field name="name">POS Bank Account</field>
```
Comment (mdi-odoo) [inline addons/pos_mercury/views/pos_mercury_views.xml]: Follow the guidelines for naming conventions.

File: addons/account_check_printing/views/account_journal_views.xml [hunk 4]
```diff
@@ -36,4 +36,17 @@
                 </xpath>
             </field>
         </record>
+        <record id="view_account_bank_journal_form_inherited" model="ir.ui.view">
```
Comment (mdi-odoo) [inline addons/account_check_printing/views/account_journal_views.xml]: Why you need this view? There is already a view for **account.journal**. See this: https://github.com/odoo/odoo/pull/15436/files#diff-d87c26a4a6f91e4d37f7ec1072df44b3R24 .

File: addons/account/models/account.py [hunk 5]
```diff
@@ -227,12 +227,14 @@ def _default_outbound_payment_methods(self):
...
+             "Batch Deposit: Encash several customer checks at once by generating a batch deposit to submit to your bank. When encoding the bank statement in Odoo,you are suggested to reconcile the transaction with the batch deposit.To enable batch deposit,module account_batch_deposit must be installed")
     outbound_payment_method_ids = fields.Many2many('account.payment.method', 'account_journal_outbound_payment_method_rel', 'journal_id', 'outbound_payment_method',
         domain=[('payment_type', '=', 'outbound')], string='Payment Methods', default=lambda self: self._default_outbound_payment_methods(),
-        help="Means of payment for sending money. Odoo modules offer various payments handling facilities, "
-             "but you can always use the 'Manual' payment method in order to manage payments outside of the software.")
+        help="Manual:Pay bill by cash or any other method outside of Odoo.\n"\
+             "Check:Pay bill by check and print it from Odoo.\n"\
+             "SEPA Credit Transfer: Pay bill from a SEPA Credit Transfer file you submit to your bank.To enable sepa credit transfer,module account_sepa must be installed")
```
Comment (fgi-odoo) [inline addons/account/models/account.py]: SEPA Credit Transfer: Pay bill from a SEPA Credit Transfer file you submit to your bank. Enable this option from the settings.

File: addons/account/models/account.py [hunk 6]
```diff
@@ -227,12 +227,14 @@ def _default_outbound_payment_methods(self):
 
     inbound_payment_method_ids = fields.Many2many('account.payment.method', 'account_journal_inbound_payment_method_rel', 'journal_id', 'inbound_payment_method',
         domain=[('payment_type', '=', 'inbound')], string='Debit Methods', default=lambda self: self._default_inbound_payment_methods(),
-        help="Means of payment for collecting money. Odoo modules offer various payments handling facilities, "
-             "but you can always use the 'Manual' payment method in order to manage payments outside of the software.")
+        help="Manual: Get paid by cash, check or any other method outside of Odoo.\n"\
+             "Electronic: Get paid automatically through a payment acquirer by requesting a transaction on a card saved by the customer when buying or subscribing online (payment token).\n"\
+             "Batch Deposit: Encash several customer checks at once by generating a batch deposit to submit to your bank. When encoding the bank statement in Odoo,you are suggested to reconcile the transaction with the batch deposit.To enable batch deposit,module account_batch_deposit must be installed")
```
Comment (fgi-odoo) [inline addons/account/models/account.py]: Batch Deposit: Encash several customer checks at once by generating a batch deposit to submit to your bank. When encoding the bank statement in Odoo,you are suggested to reconcile the transaction with the batch deposit. Enable this option from the settings.

**Pull request merged**